docs/guides/hedgehog-engine/blueblur/files/xncp/anims.md

```diff
@@ -71,10 +71,10 @@ Scary right? I don’t understand much of what’s going on here, but here we go
 - An index (for SubImage)
 - A colour (for ColorMask)
 
-According to the flags of this specific cast, these keyframes are supposed to control the X Position, which means that “data” actually represents the X Position of the cast in this frame.
+According to the flags of this specific cast, these keyframes are supposed to control the X Position, which means that “Data” actually represents the X Position of the cast in this frame.
 
-The **Offset** and **Offset2** fields are unknown. They seem to offset the value in the Data field, but I don’t know their actual meanings. I still don’t know how animations loop, and if that’s even defined in the XNCP. I’d assume it’s in some of these fields, but I’m not sure. Maybe the game just keeps playing the animation and the XNCP doesn’t control that?
-Anyway let’s look at the second keyframe then:
+The **Offset** and **Offset2** fields control the casts' Offset position value, where "Offset" is X, while "Offset2" is Y. They're usually used in the bouncing animations in the title screen, and things like that.

```

Comment:
Can you be a bit more specific with this title screen example? Not sure what you mean here...

Reply:
The buttons in the title screen in Generations have a bounce effect when they're clicked and it uses those values to control that. Will change it to be more specific

Reply:
Tried to be a bit more specific, hope it's enough